Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -212,6 +227,14 @@ private void updateSmartDashboard( )
     SmartDashboard.putNumber("poseEstimationY", m_poseEstimator.getEstimatedPosition( ).getY( ));
     SmartDashboard.putNumber("poseEstimationRotation", m_poseEstimator.getEstimatedPosition( ).getRotation( ).getDegrees( ));
 
+    Pose2d rawPoseSD = m_vision.getLimelightRawPose( );
+
+    if (rawPoseSD != null)
+    {
+      SmartDashboard.putNumber("rawPoseX", rawPoseSD.getX( ));
+      SmartDashboard.putNumber("rawPoseY", rawPoseSD.getY( ));
+    }
+
```

Comment:
I was just wondering, if you are logging the "rawPose" do you also need to log "rawPoseX" and "rawPoseY" or are they different? If they are the same, maybe we should just keep one or the other (or, since "poseEstimation" also has the same values, just keeping "poseEstimation" and getting rid of the "rawPose" logs).

Reply:
fixed